src/pages/Authentication/Login.jsx

```diff
@@ -11,6 +11,29 @@ export default function Login() {
 
   const navigate = useNavigate();
   const [error, setError] = useState(false);
+
+  const TestLogin=async()=>{
+    try{
+      const response = await axios.post(
+        "/api/auth/login",
+        getTestData()
+      );
+      console.log(response.data);
+      if (response.data.encodedToken) {
+        localStorage.setItem(
+          "login",
+          JSON.stringify(response.data.encodedToken)
+        );
+        setToken(true);
+        navigate("/notes");
+      }
+    }
+    catch(e){
+      setError(true)
+      navigate("/login");
+    }
```

Comment:
You can show a toast message and navigate user to "/login"

Reply:
ok, I will check that